supabase/migrations/099_anime_exchange_match_notification.sql

```
@@ -0,0 +1,202 @@
+        'follow',
+        'follow_request',
+        'anime_recommendation',
+        'broadcast',
+        'mylist_status',
+        'exchange_matched'
+    ));
+
+-- 未読取得高速化の部分インデックス
+CREATE INDEX IF NOT EXISTS notifications_unread_exchange_recipient_idx
+    ON public.notifications (recipient_id, created_at DESC)
+    WHERE type = 'exchange_matched' AND NOT read;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)